openmc/mgxs/mdgxs.py

```diff
@@ -1018,6 +1015,34 @@ def xs_tally(self):
 
         return self._xs_tally
 
+    def get_homogenized_mgxs(self, other_mgxs):
+        """Construct a homogenized MGXS with other mgxs objects.
```

Comment:
Sorry to be picky, but looks like "...other mgxs objects" should also capitalize "MGXS" to be perfectly explicit.